### PR TITLE
fix(worker-utils): check node worker properly

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**

### CODE REVIEW

File: modules/worker-utils/src/lib/worker-farm/worker-thread.ts [hunk 1]
```diff
@@ -28,7 +28,10 @@ export default class WorkerThread {
 
   /** Checks if workers are supported on this platform */
   static isSupported(): boolean {
-    return (typeof Worker !== 'undefined' && isBrowser) || typeof NodeWorker !== undefined;
+    return (
+      (typeof Worker !== 'undefined' && isBrowser) ||
+      (typeof NodeWorker !== undefined && !isBrowser)
```
Comment (ibgreen) [inline modules/worker-utils/src/lib/worker-farm/worker-thread.ts:33]: Is NodeWorker defined in the browser?

**Review by ibgreen — APPROVED** [review]

File: modules/worker-utils/src/lib/worker-farm/worker-thread.ts (hunk 1 above)
Comment (dryabinin-actionengine) [reply modules/worker-utils/src/lib/worker-farm/worker-thread.ts]: Yes it is always function. 
image

**Comment by ibgreen** [comment]
> Yes it is always function.

It shouldn't be. It makes me suspect that somehow our node.js code exclusion setup is failing and some kind of polyfill is being included. This looks like a deeper problem. 

What bundler are you using? webpack 4?

**Comment by dryabinin-actionengine** [comment]
> > Yes it is always function.
> 
> It shouldn't be. It makes me suspect that somehow our node.js code exclusion setup is failing and some kind of polyfill is being included. This looks like a deeper problem.
> 
> What bundler are you using? webpack 4?

In i3s example in loaders.gl repo - `"webpack": "^4.39.1",`

**Pull request merged**